extensionsGallery.json

```diff
@@ -3419,6 +3419,67 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "73",
```

Comment:
This will need to be updated to 75 so not to conflict with the other extensions that are currently only in insiders.

Reply:
Added as ExtensionID 75